18F/identity-idp
# LG-6205 Forgot Password Flow

[LG-6205](https://cm-jira.usa.gov/browse/LG-6205)

**Why**
We need to have a react version of the forgot password page which warns users that they will have to reenter their personal information before they can reset

**How**
Since this page is accessible through a link on the password confirm step (LG-6202) this is built off of that branch. 

**Testing Instructions**
1. Make sure "password confirm" is listed as one of the steps in `idv_enabled_steps` in `application.yml.default `
2. Login at localhost:3000/verify
3. Go to localhost:3000/verify/v2/password_confirm
4. Click on "forgot password" 
5. Observe that page matches screenshot 
6. Check that back button goes back
7. Check that reset password button goes to reset password page (see screenshot) 

Note: To keep the number of file changes to those which are relevant and simplify review, this PR has LG-6202 as its base rather than main.

**Screenshots**

Reset Password Page 
<img width="1080" alt="Screen Shot 2022-05-04 at 3 22 18 PM" src="https://user-images.githubusercontent.com/6639858/167487087-8bc9bc42-a849-427e-bf73-103920af9d0d.png">

Forgot Password Without Step Indicator
<img width="924" alt="Screen Shot 2022-05-04 at 3 21 16 PM" src="https://user-images.githubusercontent.com/6639858/167487126-f826948d-9a6c-4c58-bc76-4d0229cb8f2a.png">

Forgot Password Page With Step Indicator
<img width="670" alt="Screen Shot 2022-05-13 at 4 20 56 PM" src="https://user-images.githubusercontent.com/6639858/168383931-79e01cd5-c461-4aad-ae3a-b0b3130c9242.png">
